.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,31 +1,40 @@
-## Issue Title
-A clear and objective resume of the bug.
-
-## Describe the bug
-A clear and concise description of what the bug is.
-
-## To Reproduce
-To reproduce the error, the code must also be sent. It can be the origin of your application or an example (as long as it faithfully reproduces the error you are getting in the original application). 
-
-Depending on the case, it is also important to describe the steps to reproduce the error: 
-
-1. Go to '...'
-2. Click on '....'
-3. Scroll down to '....'
-4. See error
-
-## Expected behavior
-A clear and concise description of what you expected to happen.
-
-## Screenshots or videos
-If applicable, add screenshots or videos to help explain your problem.
-
-## Devices:
-Please complete the following information:
- - Device: [e.g. Raspberry Pi 3 model b+]
- - OS: [e.g. Linux Raspbian]
- - OS Version [e.g. 10]
- - TotalCross Version [e.g. 6.0.4]
-
-## Additional context
-Add any other context about the problem here.
+---
+name: Bug report
+about: Create a bug report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+## Issue Title
+A clear and objective resume of the bug.
+
+## Describe the bug
+A clear and concise description of what the bug is and the actual behavior.
+
+## To Reproduce
+To reproduce the error, the code must also be sent. It can be the origin of your application or an example (as long as it faithfully reproduces the error you are getting in the original application). 
+
+Depending on the case, it is also important to describe the steps to reproduce the error: 
+
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+## Expected behavior
+A clear and concise description of what you expected to happen.
+
+## Screenshots or videos
+If applicable, add screenshots or videos to help explain your problem.
```

Comment:
add **some** screenshots **and/or** videos to help **to** explain your problem.

Reply:
done 👍